docs/reference/central-configuration.md

```diff
@@ -0,0 +1,76 @@
+---
+navigation_title: Central configuration
+description: Reference documentation for the central configuration of EDOT SDKs.
+applies_to:
+  deployment:
+      ess: preview 9.1
+  stack: preview 9.1
+products:
+  - id: cloud
+  - id: observability
+  - id: kibana
+---
+
+# Central configuration for EDOT SDKs
+
+You can manage {{edot}} (EDOT) SDKs through the APM central configuration feature in the Applications UI. Changes are automatically propagated to the deployed [EDOT SDKs](./edot-sdks/index.md). Refer to [APM central configuration](docs-content://solutions/observability/apm/apm-agent-central-configuration.md) for more information.
```

Comment:
I'm not a `You can...` fan. But I'd love to discuss more if you are :). I would just start with: "Manage {{edot}} (EDOT) SDKs through the APM central..."

Reply:
Sounds better!